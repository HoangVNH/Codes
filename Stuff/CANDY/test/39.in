10
31
DDDDXXXXXXXXXVVVVVVVVVXDDVVVVVD
23
DDDDDVVVDDDDDDVVXXXXXVV
36
XXXXXXXXVVDDDXXXDDDXXXXXDDDDDDDDXXXX
42
DDXXXVVVXXXXDDDVVVVDDDDDXDDXXVVVVVXXDDDVVV
23
VVVVVVVVVVVVVDVVVVVDDDD
42
XXXXXXDDDDDDDVVXXXDDDXDDDVVDDDVXXXXXDDDXXX
25
XXXXXXXXXVVVVVXXXXXXXXXDD
48
DDDDXXXXXXXXXXXXXXXXXXDDDDDDDDDDVVVVVVVDDDDXXXXX
26
VVVVVVVDDDDDVVVVDDDDDVVVVV
30
XXXVVXXXVVVVVVXXXXXVVVVVVXXXXX
