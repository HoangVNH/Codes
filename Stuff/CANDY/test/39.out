30
21
34
41
22
41
24
47
24
28
